tests/simulation_data_api_test.py

```diff
@@ -33,8 +33,16 @@ def test_madx_calculate_bunch_parameters(fc):
 
 
 def _do(fc, api, method, data):
+    # TODO (gurhar1133): hacky fix below works but
+    # commented out does not.
     data.method = method
+    data.args = data.copy()
     return fc.sr_post(api, data)
+    # return fc.sr_post(api, PKDict(
+    #     method=method,
+    #     args=data
+    #     )
+    # )
 
```

Comment:
This can be fixed with the code below. The simulationId and simulationType are required arguments and the client adds them automatically with sendWithSimulationFields() in sirepo.js

```python
return fc.sr_post(api, PKDict(
	method=method,
	args=data,
	simulationId=data.simulationId,
	simulationType=data.simulationType,
    ))
```

Reply:
Nice! Thanks